src/app/core/models/rundown-event-type.ts

```diff
@@ -1,9 +1,10 @@
 export enum RundownEventType {
-  ACTIVATE = 'ACTIVATE',
-  DEACTIVATE = 'DEACTIVATE',
+  ACTIVATED = 'ACTIVATE',
+  DEACTIVATED = 'DEACTIVATE',
+  DELETED = 'DELETED',
   RESET = 'RESET',
-  TAKE = 'TAKE',
+  TAKEN = 'TAKE',
```

Comment:
Why should it be mapped to `RUNDOWN_DELETED`? Where does `RUNDOWN_` come from?
The name of your enum entry is called `DELETED`, so the value of that entry should be `DELETED`. No need to take it further than that.

Reply:
We will need events for deleting rundowns, segments, parts and other constructs. Would you use the same event for all of them?

Reply:
Currently, no.
My point was mainly towards that `RundownEventType` already indicates that it's for a Rundown, so adding `RUNDOWN_x` to the entry name is quite verbose and restating information already there.

But it would require that we can somehow differentiate between `RundownEventType.DELETED` and `SegmentEventType.DELETED`.

Sidenote: Why is `type` postfixed? What's wrong with just `RundownEvent`? It's an enum, so it's implicit that it's a type?